common/motion-dashboard.h

```diff
@@ -0,0 +1,67 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "output-model.h"
```

Comment:
These files should not be included here.
output-model.h should forward declare `stream_dashboard`, output-model.cpp should include the dashboard files it needs.
I guess device-model is included for `from_rgba` function. This function and all defined colors should be separated into colors.h file.